src/pages/index.js

```diff
@@ -247,7 +243,7 @@ export default function IndexPage({ pkgVersion }) {
           </div>
         </div>
         {visibleIcons?.length > 0 ? (
-          <div className="flex flex-wrap justify-center mx-auto bg-white">
+          <div className="flex flex-wrap justify-center mx-auto w-full bg-white">
```

Comment:
Still not _totally_ perfect when it comes to height - maybe we want a min-height on this or something?  But at least the width is covered with this PR 👍